Wikipedia/Code/AppDelegate.m

```diff
@@ -67,6 +67,10 @@ - (void)updateDynamicIconShortcutItems {
 #pragma mark - UIApplicationDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    NSString *lastAppVersion = NSUserDefaults.wmf_userDefaults.wmf_lastAppVersion;
+    if (!lastAppVersion) {
+        NSUserDefaults.wmf_userDefaults.wmf_lastAppVersion = WikipediaAppUtils.appVersion;
```

Comment:
`appVersion` includes the build number - is it OK to send snapshots when a beta user upgrades between builds or should it only be on major version changes?